databases/__init__.py

```diff
@@ -0,0 +1,94 @@
+from datetime import date
+from pony.orm import *
+
+
+db = Database()
+
+
+class User(db.Entity):
+    _table_ = "users"
+
+    # User ID
+    id = PrimaryKey(int, auto=True)
+    # User login
+    login = Required(str, 64, unique=True)
+    # User password
+    password = Required(str, 64)
+    # User permissions
+    # bitmask, perms start from the second number from left
+    # 1. can chat
+    # 2. unban
+    # 3-6. other
+    permissions = Required(int, size=32, default=1110000)
+    # User last online date
+    last_online = Required(date)
+    # User last IP
+    last_ip = Optional(str, 15, nullable=True)
+    state = Required(int, size=32, default=0)
+    # User state (what ???)
```

Comment:
this is necessary in order to prevent simultaneous login from multiple devices to 1 account

Reply:
last_online can be used for that. Will remove in next commit.